signal/signal.go

```diff
@@ -19,6 +22,50 @@ var (
 	started int32
 )
 
+// systemdNotifyReady notifies systemd about LND being ready, logs the result of
+// the operation or possible error. Besides logging, systemd being unavailable
+// is ignored.
+func systemdNotifyReady() error {
+	notified, err := daemon.SdNotify(false, daemon.SdNotifyReady)
+	if err != nil {
+		err := fmt.Errorf("failed to notify systemd %v (if you aren't"+
+			"running systemd clear the environment variable"+
+			"NOTIFY_SOCKET)", err)
+		log.Error(err)
+		// The SdNotify doc says it's common to ignore the
+		// error. We don't want to ingore it because if someone
+		// set up systemd to wait for initialization other
+		// processes would get stuck.
+		return err
+	}
+	if notified {
+		log.Info("Systemd was notified about our readyness")
+	} else {
+		log.Info("We're not running within systemd")
+	}
+	return nil
+}
+
+// systemdNotifyStop notifies systemd that LND is stopping and logs error if
+// the notification failed. It also logs if the notification was actually sent.
+// Systemd being unavailable is intentionally ignored.
+func systemdNotifyStop() {
+	notified, err := daemon.SdNotify(false, daemon.SdNotifyStopping)
+	// Just log - we're stopping anyway
```

Comment:
nit: add newline before comment and use full stop at end of sentences (here and in the Godoc comments below).